### ISSUE TITLE
Batch norm or Layer norm

### ISSUE THREAD

**Issue by CNwangbin** [bio-ontology-research-group/deepgozero#7]
https://github.com/bio-ontology-research-group/deepgozero/blob/a850e336856bf8030f8ffb121c8c00f237040ff6/deepgozero.py#L251
batch norm in paper but found Layer norm in code.
![image](https://user-images.githubusercontent.com/87804802/186813024-33585d92-9394-44ab-b4df-c8f9e67a8609.png)

**Comment by coolmaksat** [comment]
Hi,
It should be BatchNorm in the code, but LayerNorm also gives the same results. Thank you for reporting, I'll update it
